### DISCUSSION TITLE
Εξέταση Φεβρουαρίου (Επί Πτυχίω)

### DISCUSSION THREAD

**Discussion by p16z** [courses-ionio/sw#1025] [Announcements]
Καλησπέρα σας,

Όσον αφορά τους επί πτυχίω φοιτητές, θα ήθελα να σας ρωτήσω κάτι για το συγκεκριμένο μάθημα. Κατά πρώτον, θα ήθελα να ξέρω αν μπορώ να παραδώσω τις εργασίες του μαθήματος στην εξεταστική του Φεβρουαρίου 2022 (επί πτυχίω). Σε περίπτωση που είναι δυνατόν να τις παραδώσω, θα ήθελα να ξέρω αν γίνεται, ποιες από αυτές θα πρέπει να ετοιμάσω συγκεκριμένα, εφόσον μερικές από αυτές δεν είναι δυνατόν να υλοποιηθουν λόγω διαφόρων συνθηκών (π.χ. βίντεο κουίζ, συμμετοχή στην τάξη). Ευχαριστώ πολύ!

Με εκτίμηση,
Αλέξανδρος Ζανέ (Π2016087)

**Comment by epidrome** [comment]
duplicate #1023

**Reply by epidrome** [reply]
>  εφόσον μερικές από αυτές δεν είναι δυνατόν να υλοποιηθουν λόγω διαφόρων συνθηκών (π.χ. βίντεο κουίζ, συμμετοχή στην τάξη)

η μη-συμμετοχή ήταν δική σου επιλογή και προφανώς μπορεί να γίνει μόνο κατά την κανονική διάρκεια το μαθήματος, 

το μάθημα είναι ήδη σχεδιασμένο με έμφαση στην (κακώς λεγόμενη) απαλλακτική εργασία, αλλά ας μην πάμε στο άλλο άκρο όπου κάποιος μπορεί να περάσει χωρίς να μπει ποτέ στην (φυσική ή ψηφιακή) τάξη.

**Reply by p14pall** [reply]
@epidrome 
Καλησπέρα και από εμένα,
Παραθέτω την αναφορά μου εκ νέου για δεύτερη φορά, όσο καλύτερα γίνεται, (δεν ειχε ολοκληρωθεί) απο έλλειψη χρόνου λόγω εργασίας.
Έχω απαντήσει σε ορισμένα video quiz στην προηγούμενη αναφορά, θα μπορούσα να τα συμπεριλάβω στο τέλος της νέας?
